ports/atmel-samd/Makefile

```diff
@@ -412,13 +412,16 @@ endif
 SRC_SHARED_MODULE_EXPANDED = $(addprefix shared-bindings/, $(SRC_SHARED_MODULE)) \
                              $(addprefix shared-module/, $(SRC_SHARED_MODULE))
 
+SRC_O = supervisor/$(CHIP_FAMILY)_cpu.o
+
```

Comment:
Same thing here that you did in the nrf makefile?

Reply:
Done! Please take another look.